### PR TITLE
fix(webpack-cli): verbose flag functionality

### LINKED ISSUE #1542: bug: --verbose is not working
**Describe the bug**

`--verbose` flag is not working.

**What is the current behavior?**

Screenshot at 2020-05-13 16-37-49

**To Reproduce**

`webpack --verbose`

**Expected behavior**
should set `stats: verbose`

**Please paste the results of `webpack-cli info` here, and mention other relevant information**

**Additional context**
Looking into it.

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
fixes verbose flag implementation.

**Did you add tests for your changes?**
Yes
**If relevant, did you update the documentation?**
NA
**Summary**
Fixes #1542 

## Before 

Screenshot at 2020-05-13 16-37-49

## After

Screenshot at 2020-05-18 11-45-54

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: packages/webpack-cli/lib/groups/StatsGroup.js [hunk 1]
```diff
@@ -15,14 +15,10 @@ class StatsGroup extends GroupHelper {
     resolveOptions() {
         if (this.args.verbose && this.args.stats) {
             logger.warn('Conflict between "verbose" and "stats" options. Using verbose.');
-            this.opts.option.stats = {
-                verbose: true,
-            };
+            this.opts.options.stats = 'verbose';
```
Comment (snitin315) [inline packages/webpack-cli/lib/groups/StatsGroup.js:18]: The current behavior is that passing only `--verbose` also invokes `this.args.stats`.
> `this.args` & `this.opts` for `webpack --verbose`

>Screenshot at 2020-05-16 10-01-20

Looking into it.

Comment (snitin315) [reply packages/webpack-cli/lib/groups/StatsGroup.js:18]: `this.args.stats` is always true.

Screenshot at 2020-05-16 14-14-38

File: test/verbose/verbose-flag.test.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+// eslint-disable-next-line node/no-unpublished-require
+const { run } = require('../utils/test-utils');
+
+describe('verbose flag', () => {
+    it('should accept --verbose', () => {
```
Comment (rishabh3112) [inline test/verbose/verbose-flag.test.js:6]: Need a better test here. i.e. we should test for lines in stdout that are found only in verbose stats.

Comment (snitin315) [reply test/verbose/verbose-flag.test.js:6]: updated :+1:, Please take a look.

**Pull request merged**